docs/advanced/styling.md

```diff
@@ -8,20 +8,337 @@ has_toc: false
 
 # Styling Tasks
 
-Each task entry has CSS styles that allow you to change the look and feel of how the tasks are displayed. The
-following styles are available.
+In rendered queries and Reading View, the Tasks plugin adds detailed CSS classes that represent many of each task's content, to allow for very extensive styling options via CSS.
+Not only each component in a rendered task line is tagged with classes to differentiate it, many components also add classes that represent the actual content of the task, so CSS rules can refer to data such as the relative due date of a task or its specific priority.
+
+## Basic Task Structure
+
+{: .released }
+The following description relates to a restructuring of the rendered tasks that was introduced in Tasks X.Y.Z.
+
+The Tasks plugin renders a task in the following structure (this refers to query results, but the Reading View is the same exact the top-most containers):
```

Comment:
(I know I've just done a big release so it may calm down, but lately I am getting 10-20 Tasks support emails a day, and it seems that lots of people are reading the docs... So I have a bit of a vested interest in making the docs head off likely questions before release 😢  )

Reply:
Any styles starting from the task container work both in queries and Reading View, but I'll clarify this better, in case users may want styling to *differ* between queries and Reading View with additional selectors.